docs/tools/vdb_table/data/neo4j.json

```diff
@@ -14,26 +14,27 @@
     "comment": ""
   },
   "license": {
-    "value": "",
+    "value": "Proprietary",
     "source_url": "",
     "comment": ""
   },
   "dev_languages": {
     "value": [
-      ""
+      "Java",
+      "Scala"
     ],
     "source_url": "",
     "comment": ""
   },
-  "github_stars": 0,
-  "vector_launch_year": 0,
+  "github_stars": 12200,
+  "vector_launch_year": 2023,
   "metadata_filter": {
     "support": "full",
     "source_url": "",
     "comment": ""
   },
   "hybrid_search": {
-    "support": "none",
+    "support": "full",
```

Comment:
please provide a reference to docs/example notebook for this in "source_url". It should show ability to merge between ranked lists of keyword search and vector search results.